prompt-service/src/unstract/prompt_service/helper.py

```diff
@@ -286,8 +300,17 @@ def run_completion(
         )
         highlight_data = None
         if highlight_data_plugin and enable_highlight:
+            fs_instance: FileStorage
```

Comment:
So does this mean that the tool instance check is not working?

Reply:
@gaya3-zipstack Yes, while checking in the code, the `PromptIdeBaseTool` ideally should have been in SDK for this check to work.